sycl/include/sycl/device_aspect_macros.hpp

```diff
@@ -691,11 +686,6 @@
 #define __SYCL_ANY_DEVICE_HAS_ext_oneapi_graph__ 0
 #endif
 
-#ifndef __SYCL_ANY_DEVICE_HAS_ext_intel_fpga_task_sequence__
-// __SYCL_ASPECT(ext_intel_fpga_task_sequence__, 62)
-#define __SYCL_ANY_DEVICE_HAS_ext_intel_fpga_task_sequence__ 0
-#endif
-
 #ifndef __SYCL_ANY_DEVICE_HAS_ext_oneapi_limited_graph__
 // __SYCL_ASPECT(ext_oneapi_limited_graph, 63)
```

Comment:
This and the following defines should all have been updated, similar to `ALL_DEVICE_HAS`.

The ID specified for `ext_oneapi_limited_graph` now does not match the ID defined in `aspects.def` anymore. The same applies for all following defines.

@aejjehint Can you please fix this in a follow-up PR?